### PR TITLE
drivers/at: fixed faulty event prio name

### PR DESCRIPTION
**Pull request by viktorbatista**
Fixed a small oversight if using the at_urc_isr_highest pseudomodule.

### Contribution description

Basically just fixes an oversight in the at driver where it was checking if the at_urc_isr_high module was defined, which doesn't exist - it should have been at_urc_isr_highest. 
Just below that line it then defines AT_EVENT_PRIO as EVENT_PRIO_HIGH which also should have been EVENT_PRIO_HIGHEST.

### Testing procedure

Just include the at_urc_isr_highest module anywhere in a build and notice how it no longer outputs an error ;)

### Issues/PRs references

-

### CODE REVIEW

**Review by benpicco — APPROVED** [review]

**Comment by benpicco** [comment]
bors merge

File: drivers/at/at.c [hunk 1]
```diff
@@ -25,11 +25,11 @@
 #endif
 
 #if defined(MODULE_AT_URC_ISR_LOW)
```
Comment (benpicco) [inline drivers/at/at.c:27]: This should then also be called `MODULE_AT_URC_ISR_LOWEST` (even though `at_urc_isr_low` *does* exist).

Comment (viktorbatista) [reply drivers/at/at.c:27]: Alright, I'll change it real quick

**Review by maribu — APPROVED** [review]

**Comment by maribu** [comment]
Thx for spotting and fixing this!

**Comment by viktorbatista** [comment]
bors merge

**Comment by benpicco** [comment]
bors merge

**Pull request merged**